morpheus/_lib/src/messages/multi_response.cpp

```diff
@@ -66,7 +74,8 @@ std::shared_ptr<MultiResponseMessage> MultiResponseMessageInterfaceProxy::init(s
 
 std::shared_ptr<morpheus::ResponseMemory> MultiResponseMessageInterfaceProxy::memory(MultiResponseMessage &self)
 {
-    return DCHECK_NOTNULL(std::dynamic_pointer_cast<morpheus::ResponseMemory>(self.memory));
+    DCHECK(std::dynamic_pointer_cast<morpheus::ResponseMemory>(self.memory) != nullptr);
+    return std::static_pointer_cast<morpheus::ResponseMemory>(self.memory);
```

Comment:
```suggestion
    DCHECK(std::dynamic_pointer_cast<morpheus::ResponseMemory>(self.memory) != nullptr);

    return std::static_pointer_cast<morpheus::ResponseMemory>(self.memory);
```
Match the spacing of the others